.gitignore

```diff
@@ -25,6 +25,7 @@ profile.html
 ## Environment normalisation:
 /.bundle/
 /lib/bundler/man/
+dev.yml
```

Comment:
Leftover from testing?

Reply:
no, just like other editor files, I intend to add this, you will see it in paquito and other open source libs as well... it makes it a bit easier to work with some of our standard tooling